lib/1.2/dml-builtins.dml

```diff
@@ -5,6 +5,7 @@
 
 dml 1.2;
 
+provisional c_vect;
```

Comment:
Why? Is it because you may have a call to an inline method of dml-builtins reference vects? That feels like a shortcoming of the provisional mechanism if so.

Reply:
Because
```
extern typedef _callback_entry_t * vect _callback_vect_t;
extern typedef _connection_entry_t * vect _connection_vect_t;
```

Reply:
ohhhhh